sdk/extendedlocation/Microsoft.Azure.Management.ExtendedLocation/src/Microsoft.Azure.Management.ExtendedLocation.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.ExtendedLocation</PackageId>
+    <Description>Provides Microsoft Azure ExtendedLocation operations.</Description>
+    <Version>1.0.0</Version>
```

Comment:
You should start out as a prerelease project, I think, so version should be '1.0.0-beta.1'

Reply:
What would classify a Project as a prelease project?